src/index.jsx

```diff
@@ -69,22 +56,41 @@ function configure() {
   };
 }
 
-apiClient.ensurePublicOrAuthenticationAndCookies(
-  window.location.pathname,
-  (accessToken) => {
-    const { store, history } = configure();
+function initialize(accessToken) {
```

Comment:
This function is just a named version of the callback/closure we used to give to `ensurePublicOrAuthenticationAndCookies`.